ts-declarations/xcode/index.d.ts

```diff
@@ -0,0 +1,48 @@
+declare module 'xcode' {
```

Comment:
I've moved the splash-screen variation to this location https://github.com/expo/expo-cli/pull/2592 and refactored config to use it